rpm-build/SPECS/securedrop-workstation-dom0-config.spec

```diff
@@ -1,16 +1,28 @@
-Name:		securedrop-workstation-dom0-config
-Version:	0.5.3
+%global srcname securedrop-workstation-dom0-config
+%global version 0.5.3
+%global __python3 /usr/bin/python3
+%global python3_sitelib /usr/lib/%{_python_version}/site-packages
+# For reproducible builds
+%global _buildhost %{srcname}
+%global _source_date_epoch %{getenv:SOURCE_DATE_EPOCH}
+%global _custom_docdir /usr/share/doc/%{srcname}
+%define optflags -O2 -g
+
+Name:		%{srcname}
+Version:	%{version}
 Release:	1%{?dist}
 Summary:	SecureDrop Workstation
 
 Group:		Library
 License:	GPLv3+
 URL:		https://github.com/freedomofpress/securedrop-workstation
-Source0:	securedrop-workstation-dom0-config-0.5.3.tar.gz
+Source0:	%{srcname}-%%{version}.tar.gz
 
 BuildArch:      noarch
-BuildRequires:	python3-setuptools
-BuildRequires:	python3-devel
+# Disable declaration of build dependencies, because
+# we build on Debian stable.
+#BuildRequires:	python3-setuptools
```

Comment:
I am guessing this is why we add install-deps

Reply:
Precisely. I believe we could technically pass in a macro here via the shell wrapper, based on distro, but that much indirection sounds confusing, at least at this stage in our pursuit of reproducible builds.